indico/modules/events/contributions/util.py

```diff
@@ -198,6 +199,11 @@ def get_contribution_ical_file(contrib):
     serializer = Serializer.create('ics')
     return BytesIO(serializer(data))
 
+def is_valid_email(email):
+    if len(email) > 7:
```

Comment:
Why `>7`? `a@b.cc` could be a perfectly valid email address and it's only 6 characters.